kubemarine/system.py

```diff
@@ -48,30 +48,33 @@ def verify_inventory(inventory, cluster):
 
 
 def enrich_etc_hosts(inventory, cluster):
+# enrich only etc_hosts_generated object, etc_hosts remains as it is
+
     control_plain = inventory['control_plain']['internal']
 
+    # take custom name of control_plain from etc_hosts, if any
     control_plain_names = inventory['services']['etc_hosts'].get(control_plain, [])
```

Comment:
Looks like we should take custom name for control_plane from `services.etc_hosts` if it's present there. If there is none, we don't take it and proceed with default set of node names.

Reply:
If it is present there, it will be added to /etc/hosts by system.generate_etc_hosts_config(). It will not be added to `coredns.configmap.Hosts`, but we decided to make them independent.